src/CONST.ts

```diff
@@ -2705,7 +2705,8 @@ const CONST = {
         BUTTON: 'button',
         LINK: 'link',
         MENUITEM: 'menuitem',
-        TEXT: 'presentation',
+        PRESENTATION: 'presentation',
+        TEXT: 'text',
```

Comment:
Should this come in as part of this PR?

Reply:
The change to text needs to be there to prevent a crash, but not the new PRESENTATION key. The thing is this change *will* be breaking for other components until my ACCESSIBILITY_ROLE PR gets merged.
```suggestion
        TEXT: 'text',
```

Reply:
I'm leaning towards we should remove these. Your PR looks like it's dealing with a lot of merge conflicts, and I'd rather deal with type errors here than a bunch of merge conflicts later :) Thoughts?

Reply:
If we remove this the app will crash.